src/api/notification/services/notification.ts

```diff
@@ -8,70 +8,83 @@ const MODULE_ID = 'api::notification.notification'
 const GLOBAL_MODULE_ID = 'api::notifications-consumer.notifications-consumer'
 const SINGLETON_ID = 1
 
+const NOTIFICATIONS_POPULATE = {
+  notification_template: {
+    fields: ['id', 'title', 'description', 'url', 'push'],
+    populate: {
+      thumbnail: {
+        fields: ['url']
+      }
+    }
+  }
+}
+
 export default factories.createCoreService(MODULE_ID, ({ strapi }) => {
   return {
+    async getNotificationsForAll(push: boolean) {
+      return strapi.entityService.findMany(
+        MODULE_ID,
+        {
+          start: 0,
+          limit: 50,
```

Comment:
what's the plan for the hardcoded 50

Reply:
I would even keep it as it is.
We could add pagination and so on, but...will we really need it?
I think the main use-case of the notifications is to notify users about something new which is relevant only for the nearest feature.
As a user I would read even only the last 3-5 notifications.
But this is only my assumption, I'm open for any proposal.

Reply:
Okay, lets keep it for now them